app/views/admin/dashboards/show.html.erb

```diff
@@ -22,3 +22,52 @@
     <%= render partial: 'padmin', locals: {charity_type: @offline_charities} %>
 
 <% end %>
+
+
+<div class="details">
+  <% if @charity %>
+    <h2>Administrators for <%= @charity.name %></h2>
+  <% else %>
+    <h2>All Charity Administrators</h2>
+  <% end %>
+</div>
+
+<div class="search-font filter">
+    <label>Filter By Username  </label>
+    <input type="text" placeholder="Username" id="user_filter_username">
+</div>
+
+<div class="box">
+  <%= link_to "Create New Admin", new_admin_user_path, class: "btn button-indigo" %>
+</div>
+
+<div class="col-sm-12 col-sm-offset-1 users">
+  <% @users.each do |user| %>
```

Comment:
As we are dealing with specifically admins here we should make sure that we change this variable name to admin as opposed to user. It will be more clear to someone reading the code later that we are specifically displaying only admins in this section of the page.